src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -348,6 +368,16 @@ def perform_post_processing(
             # For w2, we need to post-process to allow for SSN collisions in the data in cases where
             #  the simulant has no SSN but is employed (they'd need to have supplied an SSN to their employer)
             obs_data = do_collide_ssns(obs_data, maps["simulant_id"]["ssn"], randomness)
+        if observer in [
```

Comment:
Why does this need to be outside of the main loop in lines 359-365?

Reply:
In the main loop we are generating and then applying all our maps. We need the full ssn map to be created before we can map them to "copy_ssn". It is very similar logic to the ssn collision right above, we need to post-process the ssn map which we just can do inside the main loop.